yb-voyager/src/utils/commonVariables.go

```diff
@@ -26,6 +26,7 @@ const (
 	TABLE_MIGRATION_IN_PROGRESS
 	TABLE_MIGRATION_DONE
 	TABLE_MIGRATION_COMPLETED
+	YB_VOYAGER_NULL_STRING = "__$#__YBVOYAGER_NULL__$#__"
```

Comment:
I understand why you might want to have a string like that. But this can (in certain cases) blow up the snapshot file size.

A similar but smaller alternative is `__YBV_NULL__`. This won't cause any conflict even if there is a string field containing this "keyword" -- because the string field will be quoted whereas the `__YBV_NULL__` will appear without any quotes.